docs/spec/sources.md

```diff
@@ -362,5 +376,40 @@ _Structured Content Mode_ of the HTTP Protocol Binding for CloudEvents. Sources
 SHOULD send events to its
 [Destination](https://pkg.go.dev/github.com/knative/pkg/apis/duck/v1?tab=doc#Destination).
 
+### CloudEvent `subscription` extension
+
+Sources SHOULD produce CloudEvents with the `subscription` CloudEvent extension
+identifying the originating [Source Custom Object](#source-custom-objects).
+
+The `subscription` value SHOULD be of the form `<name>.<namespace>.<plural>.<group>`, where:
```

Comment:
this is still wrong. the name of the resource is not the subscription, it is the uid of the resource and maybe a generation for that resource.

Reply:
Can we guarantee that from an end-user perspective the source (cr) will always be created before and in a distinct step from the creation of a trigger/filter?  If so, then uid might be ok - but some user-friendly info would sure be a lot nicer UX. I'm not sure why we're so concerned about using the name here when the rest of Kube typically does refs by name and not uid.

Not sure about generation though since I wouldn't want an update to a source (cr) to cause everyone to recreate their filters and lose events in the meantime.